src/server/dflycmd.cc

```diff
@@ -141,6 +138,11 @@ void DflyCmd::Run(CmdArgList args, ConnectionContext* cntx) {
     return ReplicaOffset(args, cntx);
   }
 
+  if (sub_cmd == "LOAD" && args.size() == 2) {
+    DebugCmd debug_cmd{sf_, cntx};
```

Comment:
because "debug load" is a debug, unofficial command for tests etc. 
I would like to introduce an official command so that people could actually load into dragonfly snapshots. it's actually pretty cumbersome to do it today because loading an existing snapshot demands changing `dir` or `dbfilename` but then subsequent SAVE overwrites the loaded file.

Reply:
I think that deserves to be in the PR title :)